### DISCUSSION TITLE
Questions about profile icons - is there a list of them? Is customization (such as color) possible?

### DISCUSSION THREAD

**Discussion by tv21** [Eugeny/tabby#9880] [Q&A] [answered]
I see you can set icons for each profile but is there a list of all the possible icons anywhere?  And can they be customized?  It would be nice to be able to use full color icons, not just black and white ones.

**Comment by re99io** [comment]
https://fontawesome.com/search?o=r&m=free

**Comment by tv21** [comment]
Thanks for that, but those all seem to be monochrome icons, I was hoping there was a way you could use full color icons.  They are so small anyway that sometimes it's not easy to tell them apart.  That said they do seem to have an icon for nearly everything!

**Reply by Eugeny** [reply]
It's undocumented, but you can straight up paste SVG into the icon field

**Reply by tv21** [reply]
> It's undocumented, but you can straight up paste SVG into the icon field

Maybe I'm dumb but in Ubuntu 24.04 I just can't make it work. I tried dragging and dropping an image, copying and pasting (it gave the path to the file starting with file:/// ), removing the file:// and just leaving the path, and even tried a http:// link to a .svg image but nothing seemed to work. I also tried a couple different .svg images. Tabby version is 1.0.211.

**Reply by Eugeny** [reply, marked as answer]
Nono, I mean the literal file content, SVG "code". It's a workaround that's interally used for distro-specific icons on WSL

**Reply by tv21** [reply]
Thank you, that works!  I load the .svg file into a TEXT editor, then copy and paste the text into Tabby's icon field.  Not every .svg file works but I found ones that do.  I had the best luck with .svg icons from Wikipedia.
